ChangeLog

```diff
@@ -1,3 +1,12 @@
+mtbl (1.7.0)
+
+  * Add public mtbl_threadpool API; see mtbl_threadpool manpage for more info.
```

Comment:
maybe add a summary of the feature at the top hear.  From your MR: "Add multithreading functionality to mtbl_writer and mtbl_sorter"

Reply:
Pushed